mm2src/coins/lp_coins.rs

```diff
@@ -3676,6 +3680,9 @@ impl DexFee {
             Ok(None)
         }
     }
+
+    /// Check and return true if DexFee is not required to trade otherwise return false.
+    pub fn no_fee(&self) -> bool { matches!(self, Self::Zero) }
```

Comment:
```suggestion
    /// Check and return true if DexFee is not required to trade otherwise return false.
    pub fn zero_fee(&self) -> bool { matches!(self, Self::Zero) }
```

Reply:
done